src/pages/ReportDetailsPage.js

```diff
@@ -87,12 +87,24 @@ class ReportDetailsPage extends Component {
         }
 
         const policy = this.props.policies[`${ONYXKEYS.COLLECTION.POLICY}${this.props.report.policyID}`];
-        if (ReportUtils.isUserCreatedPolicyRoom(this.props.report) || ReportUtils.canLeaveRoom(this.props.report, !_.isEmpty(policy))) {
+        if (
+            (ReportUtils.isUserCreatedPolicyRoom(this.props.report) || ReportUtils.canLeaveRoom(this.props.report, !_.isEmpty(policy))) &&
+            _.isEmpty(this.props.report.parentReportActionID)
+        ) {
             menuItems.push({
                 key: CONST.REPORT_DETAILS_MENU_ITEM.LEAVE_ROOM,
                 translationKey: 'common.leaveRoom',
                 icon: Expensicons.Exit,
-                action: () => Policy.leaveRoom(this.props.report.reportID),
+                action: () => Report.leaveRoom(this.props.report.reportID),
+            });
+        }
+
+        if (this.props.report.parentReportActionID) {
```

Comment:
once https://github.com/Expensify/App/pull/18522 is merged we can replace with `ReportUtils.isThread(report)` for readibility.